mount/mount_windows.go

```diff
@@ -17,17 +17,48 @@
 package mount
 
 import (
+	"context"
 	"encoding/json"
+	"errors"
 	"fmt"
 	"os"
 	"path/filepath"
 	"strings"
 
+	"github.com/Microsoft/go-winio/pkg/bindfilter"
 	"github.com/Microsoft/hcsshim"
+	"github.com/containerd/containerd/log"
+	"golang.org/x/sys/windows"
 )
 
+const sourceStreamName = "containerd.io-source"
+
+var (
+	// ErrNotImplementOnWindows is returned when an action is not implemented for windows
+	ErrNotImplementOnWindows = errors.New("not implemented under windows")
+)
+
+func (m *Mount) ReadOnly() bool {
+	for _, option := range m.Options {
+		if option == "ro" {
+			return true
+		}
+	}
+	return false
+}
+
 // Mount to the provided target.
-func (m *Mount) mount(target string) error {
+func (m *Mount) mount(target string) (retErr error) {
+	if m.Type == "bind" {
+		if !strings.EqualFold(filepath.Base(m.Source), "Files") {
```

Comment:
@TBBle I believe this is what you meant. Lemme know if I misread :smile:

Reply:
Heh, we crossed comments: https://github.com/containerd/containerd/pull/8043/files/4012c1b853adf637667ad62c74c3fa61b3f5e22e..b837c141def583d3e7add3015bd16623ed28e221#r1154603885